torchbox/wagtail-torchbox
# Remove Service & SubService Pages

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1220423226)

### Description of Changes Made

With the change from ServicePages to PropositionPages, and SubServicePages to SubPropositionPages, we no longer need the models in the `services` app. This MR is to remove them from the codebase.

**Caveats**

- Unfortunately, we cannot completely delete the `services` app, as it contains migrations that are depended upon by other apps. This means we cannot also remove it from the `INSTALLED_APPS` list. 

### How to Test

- when you run migrations against `staging` / `production` / a fresh database, there should be no errors.
- in Wagtail admin, when creating a new Page, there should be no option to add a ServicePage or SubServicePage  

### MR Checklist
